src/net/i2p_address.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2018, The Monero Project
```

Comment:
2019

Reply:
OK, well I started this in 2018, but changed anyway.